packages/statemanager/test/statelessVerkleStateManager.spec.ts

```diff
@@ -43,4 +52,115 @@ describe('StatelessVerkleStateManager: Kaustinen Verkle Block', () => {
       'should have correct codeHash'
     )
   })
+
+  it('put/delete/modify account', async () => {
+    const stateManager = new StatelessVerkleStateManager({ common })
+    stateManager.initVerkleExecutionWitness(block.executionWitness)
+
+    const address = new Address(randomBytes(20))
+
+    let check = await stateManager.getAccount(address)
+    assert.isUndefined(check, 'should return undefined for nonexistent account')
+
+    const account = Account.fromAccountData({
+      nonce: BigInt(2),
+    })
+    await stateManager.putAccount(address, account)
+
+    check = await stateManager.getAccount(address)
+    assert.deepEqual(check, account, 'should return correct account')
+
+    await stateManager.modifyAccountFields(address, {
+      nonce: BigInt(2),
+    })
+    check = await stateManager.getAccount(address)
+    assert.deepEqual(check, account, 'should return correct account')
+
+    await stateManager.deleteAccount(address)
+
+    check = await stateManager.getAccount(address)
+    assert.isUndefined(check, 'should return undefined for deleted account')
```

Comment:
Again, this will throw. We could modify the tests so that we expect throw. 

Could also be nice to add a test case for the following scenario:

- Account A does not exit
- `null` witness is provided for non-existent account
- getAccount(accountA) should return undefined

Reply:
I've modified the test to catch the initial thrown error when getting a random address that doesn't exist in the witness, but this assert after a deleted account is got actually is not throwing but returning an `undefined`, currently. 

I will work on more test cases including the suggested one in future PR's as I'm still working on collecting witness data for it.